src/PKSim.UI/Views/Parameters/MultiParameterEditView.cs

```diff
@@ -239,11 +239,32 @@ public void FixParameterColumnWidth(int parameterWitdh)
          _columnValue.WithFixedWidth(parameterWitdh);
       }
 
-      public IEnumerable<ParameterDTO> SelectedParameters => gridViewParameters.DataController.GetAllFilteredAndSortedRows().Cast<ParameterDTO>();
+      public IEnumerable<ParameterDTO> AllVisibleParameters => gridView.DataController.GetAllFilteredAndSortedRows().Cast<ParameterDTO>();
+
+      public IReadOnlyList<ParameterDTO> SelectedParameters
+      {
+         get { return gridView.GetSelectedRows().Select(rowHandle => _gridViewBinder.ElementAt(rowHandle)).ToList(); }
+         set
+         {
```

Comment:
Little bit hugly here to select rows programmatically. I need to find the row handles , select the rows and then set the focus row handle. Does not work if rows are not continuously selected but this is not possible in this view